src/pages/Search/Search.js

```diff
@@ -1,23 +1,61 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import { useParams } from 'react-router-dom';
+import config from '../../config';
 import Page from '../../components/Page';
-import HeatMap from '../../components/HeatMap';
-import ResultsTable from '../../components/ResultsTable';
 import InputForm from '../../components/InputForm';
 import Headline from '../../components/Headline';
-
+import Spinner from '../../components/Spinner';
 
 export default function Search() {
+  const { slug } = useParams();
+
+  const [data, setData] = useState([]);
+  const [loading, toggleLoading] = useState(false);
+  const [errorMessage, setError] = useState('');
+
+  let count = 0;
+  let after = '';
+  const loopData = [];
+
+  const fetchData = () => {
+    const url = `${config.api_url}/r/${slug}.json?&after=${after}&limit=100`;
```

Comment:
This URL is not correct. I think it will simply get the latest posts in a subreddit. But we need the top posts of the last year. This is the correct one

```
https://www.reddit.com/r/${subredditName}/top.json?t=year&limit=100
```